x/ccv/provider/client/proposal_handler.go

```diff
@@ -74,6 +75,16 @@ Where proposal.json contains:
 				return err
 			}
 
+			// do not fail for errors regarding the unbonding period, but just log a warning
+			err = CheckPropUnbondingPeriod(clientCtx, proposal.UnbondingPeriod)
+			if err != nil {
+				fmt.Fprintf(
```

Comment:
Hey, I'm bikeshedding but why not just do this?
```suggestion
fmt.Errorf("Warning: Could not assure that Proposal Unbonding Period is shorter than Provider Unbonding Period. Error message: %s, err.Error()
```

Reply:
Not clear what you mean by this. What you wrote creates an error object, but I want to primt to Stderr.